### ISSUE TITLE
vmauth: add ability to specify default route in url_map

### ISSUE THREAD

**Issue by valyala** [VictoriaMetrics/VictoriaMetrics#4084]
### Is your feature request related to a problem? Please describe

Currently [vmauth](https://docs.victoriametrics.com/vmauth.html) allows configuring fine-grained access to different backend via `url_map` section. For example, the following config proxies `/foo/bar` requests from `foobar` user to `http://backend1234/abc/foo/bar` backend url:

```yml
users:
- username: "foobar"
  url_map:
  - src_paths:
    - /foo/bar
    url_prefix: "http://backend123/abc"
```

Requests to all the other paths are rejected with non-200 HTTP response. For example, if `foobar` user requests `/some/other/path`, then it receives non-200 HTTP response. Sometimes it is needed to proxy all such requests to some backend url for further processing or for providing custom response page.

### Describe the solution you'd like

It would be great to have the ability to specify backend url for requests, which do not match `src_paths` from the `url_map`. This could be made via `default_url` option. For example, the following config could proxy all the requests other than `/foo/bar` to `http://fail-backend/abc`:

```yml
users:
- username: "foobar"
  url_map:
  - src_paths:
    - /foo/bar
    url_prefix: "http://backend123/abc"
  default_url: "http://fail-backend/abc"
```

The original request path shouldn't be appended to the `default_url`. Instead, it could be passed in `request_path` query arg, so the backend could analyze which paths were requested.

### Describe alternatives you've considered

To configure an additional proxy such as `nginx` in front of `vmauth`, with could route requests to unsupported paths to `http://fail-backend/abc`.

### Additional information

cc'ing @f41gh7 , since he is interested in this feature.

**Comment by valyala** [comment]
The `default_url` option is available in `vmauth` starting from the commit 041e188df856ddfc80477d73b73c2a26c901aefb . This commit will be included in the next release.

**Comment by valyala** [comment]
The `default_url` option is available in [vmauth](https://docs.victoriametrics.com/vmauth.html) starting from [v1.91.0](https://docs.victoriametrics.com/CHANGELOG.html). Closing the feature request as done.
